config/wraith.yaml

```diff
@@ -0,0 +1,55 @@
+---
+browser:
+  phantomjs: phantomjs
+directory: shots
+screen_widths:
+- 1024
+mode: diffs_only
+fuzz: 5%
+threshold: 5
+num_threads: 4
+timeout_ms: 2000
+paths:
```

Comment:
Updated the Technical documentation of the readme file to include:

When `rake diff` is run, a list of all the document formats on govuk is
obtained using the search api, and then the top n pages for each format (n
being a configuration variable).  Diffs are produced for each of these
pages.


On 21 December 2015 at 09:44, Tijmen Brommet <notifications@github.com>
wrote:

> In config/wraith.yaml
> <https://github.com/alphagov/govuk-diff-pages/pull/1#discussion_r48129684>
> :
>
> > @@ -0,0 +1,55 @@
> > +---
> > +browser:
> > +  phantomjs: phantomjs
> > +directory: shots
> > +screen_widths:
> > +- 1024
> > +mode: diffs_only
> > +fuzz: 5%
> > +threshold: 5
> > +num_threads: 4
> > +timeout_ms: 2000
> > +paths:
>
> How is this list of pages generated?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/govuk-diff-pages/pull/1/files#r48129684>.
>



-- 
Stephen Richards
stephen.richards@digital.cabinet-office.gov.uk